.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
```

Comment:
Out of curiosity, any reason to change this?  One of the things I was using it for was consistent formatting on commit.  Not urgent, but food for thought